site/pages/dept-en.hbs

```diff
@@ -1,78 +1,44 @@
-		<div class="form-group">
-			<div class="col-lg-3">
-				<label for="deptSearch" class="control-label mrgn-rght-lg">What are you looking for?</label>
-			</div>
-			<div class="col-lg-9 input-group">
-				<input type="text" class="form-control" size="10" id="deptSearch" placeholder="Department or agancy" />
-				<span class="input-group-btn">
-		            <button type="submit" id="search-submit" class="btn btn-primary" name="search-submit"><span class="glyphicon-search glyphicon"></span><span class="wb-inv">Search departments & agencies</span></button>
-		        </span>
-			</div>
-		</div>
-		</form>
-	</div>
+ <div class="row profile">
+  <div class="col-md-12">
+    <p class="pagetag">Enter a department name or abbreviation to find current Government of Canada departments, agencies, crown corporations and special operating agencies.</p>
+  </div>
 </div>
-<hr />
-<ul class="list-unstyled lst-spcd">
-	<li><a class="h3" href="#">Department name</a><br />
-		<span class="badge badge-dept">8</span>Also works with…
-		<ul class="mrgn-bttm-lg mrgn-lft-lg">
-			<li><a href="#">Institution name</a></li>
-			<li><a href="#">Institution name</a></li>
-			<li><a href="#">Institution name</a></li>
-			<li><a href="#">Institution name</a></li>
-			<li><a href="#">Institution name</a></li>
-			<li><a href="#">Institution name</a></li>
-			<li><a href="#">Institution name</a></li>
-			<li><a href="#">Institution name</a></li>
-		</ul>
-	</li>
-	<li><a class="h3" href="#">Department name</a><br />
-		<span class="badge badge-dept">6</span> Also works with…
-		<ul class="mrgn-bttm-lg mrgn-lft-lg">
-			<li><a href="#">Institution name</a></li>
-			<li><a href="#">Institution name</a></li>
-			<li><a href="#">Institution name</a></li>
-			<li><a href="#">Institution name</a></li>
-			<li><a href="#">Institution name</a></li>
-			<li><a href="#">Institution name</a></li>
-		</ul>
-	</li>
-	<li><a class="h3" href="#">Department name</a><br />
-		<span class="badge badge-dept">16</span>Also works with…
-		<ul class="mrgn-bttm-lg mrgn-lft-lg">
-			<li><a href="#">Institution name</a></li>
-			<li><a href="#">Institution name</a></li>
-			<li><a href="#">Institution name</a></li>
-			<li><a href="#">Institution name</a></li>
-			<li><a href="#">Institution name</a></li>
-			<li><a href="#">Institution name</a></li>
-			<li><a href="#">Institution name</a></li>
-			<li><a href="#">Institution name</a></li>
-			<li><a href="#">Institution name</a></li>
-			<li><a href="#">Institution name</a></li>
-			<li><a href="#">Institution name</a></li>
-			<li><a href="#">Institution name</a></li>
-			<li><a href="#">Institution name</a></li>
-			<li><a href="#">Institution name</a></li>
-			<li><a href="#">Institution name</a></li>
-			<li><a href="#">Institution name</a></li>
-		</ul>
-	</li>
-</ul>
\ No newline at end of file
+<table class="wb-tables table" data-wb-tables="{ &quot;paging&quot;: false }">
```

Comment:
Now we have a "filter" plugin, I would recommend to use it for that page instead of relying on the data table search feature.  Make that suggestion to the policy, they might now aware of that plugin http://wet-boew.github.io/wet-boew/demos/filter/filter-en.html